app/src/main/res/values/strings.xml

```diff
@@ -403,4 +409,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_address_house_name_label">House name:</string>
     <string name="quest_buildingLevels_roofLevelsLabel2">levels in the roof</string>
     <string name="quest_buildingLevels_levelsLabel2">regular levels (not counting the roof)</string>
+    <string name="quest_range1">0</string>
```

Comment:
If the quest range question type is kept, these should be named more specifically. It's unlikely that most range questions will always fit into the categories of 0, 1-3, and 4-7, so I'd call it quest_range_lit_1 or something.